.changeset/fast-moles-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
+---
+
+Add `sshd` to codespace container
```

Comment:
This doesn't need a changeset since it's not a consumer facing change

Reply:
👍🏻